ldapauth/README.md

```diff
@@ -0,0 +1,49 @@
+Authenticate a user against an LDAP directory
+===
+
+Useful for Windows Active Directory and other LDAP-based organisations
```

Comment:
Could you please stick to the "one sentence per line" rule in Markdown files. It makes the diffs more usable and MD files are rendered anyway. It also helps to prevent very long sentences and thus makes the docs more read-able.

Reply:
Not if it's the only change you are requesting. I agree with you on principle but this PR wasn't meant to completely reformat the addons' doc. I didn't even read the offending paragraphs, I just added a title and changed the config style.